### PR TITLE
[Remove Blocking usages]LPS-189564 portal-kernel | WorkflowComparatorFactoryUtil 

### PR DESCRIPTION
**Pull request by JanisZhang**
resend, previous: https://github.com/liferay-core-infra/liferay-portal/pull/5086

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:test:startup

**Comment by JanisZhang** [comment]
ci:test:workflow

**Comment by JanisZhang** [comment]
ci:stop

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:startup

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by JanisZhang** [comment]
ci:test:workflow

File: modules/apps/portal-workflow/portal-workflow-api/src/main/java/com/liferay/portal/workflow/comparator/WorkflowComparatorFactory.java [hunk 1]
```diff
@@ -1,9 +1,9 @@
 /**
- * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
```
Comment (dantewang) [inline modules/apps/portal-workflow/portal-workflow-api/src/main/java/com/liferay/portal/workflow/comparator/WorkflowComparatorFactory.java:2]: @JanisZhang Please revert this change and force push, thanks!

**Comment by Kvale1733** [comment]
@JanisZhang No regressions found.

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
ci:forward:force

cc @tinatian

**Comment by dantewang** [comment]
@JanisZhang My bad, it seems it's modified by SF... Could you force push again? Thanks!

**Comment by dantewang** [comment]
ci:stop

**Comment by dantewang** [comment]
ci:stop

**Comment by JanisZhang** [comment]
Hi @dantewang, I have force-pushed. I only changed the year in the copyright. it shouldn't affect the test results. forwarding this ....  cc @Kvale1733

**Comment by JanisZhang** [comment]
ci:forward:force

**Comment by dantewang** [comment]
ci:test:stable

**Comment by dantewang** [comment]
ci:forward:force

cc @tinatian
